### ISSUE TITLE
Disabling certain files causes issues with the acronyms file

### ISSUE THREAD

**Issue by petersaints** [joaomlourenco/novathesis#122]
I was in the middle of updating my thesis document to the latest template version and stumbled upon this issue. I was able to reproduce it on a completely clean version of the template, so it should not be too hard to pinpoint the cause of the issue.

On my thesis I currently don't have a Glossary, Symbols, but I have Acronyms. Therefore, I only left the acronyms line uncommented, but I got an error while doing so. 

I did the the following on a clean template with similar results:
```
% Lists of Glossary, Acronyms and Symbols
% syntax:  \ntaddfile{glossaries}[list-type]{filename}
%\ntaddfile{glossaries}[glossary]{glossary}
\ntaddfile{glossaries}[acronyms]{acronyms}
%\ntaddfile{glossaries}[symbols]{symbols}
```
Please take note that the only changes are that the lines corresponding to the parts of the document that I don't want were commented out. The error that was risen was the following:
```
K=acronyms~V=acronyms
(./Chapters/acronyms.tex
NT LOADING FILE acronyms.tex
)
K=acronyms~V=acronyms
(./Chapters/acronyms.tex
NT LOADING FILE acronyms.tex

! Package glossaries-extra Error: Glossary entry `abbrev' has already been defined.

See the glossaries-extra package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.12 ...ev}{abbrev}{abbreviation of a longer text}
                                                  
?
```

I tested other combinations, and it only works with:
```
% Lists of Glossary, Acronyms and Symbols
% syntax:  \ntaddfile{glossaries}[list-type]{filename}
\ntaddfile{glossaries}[glossary]{glossary}
\ntaddfile{glossaries}[acronyms]{acronyms}
\ntaddfile{glossaries}[symbols]{symbols}
```

If one or two of the three lines is commented you will invariably get an error that something (either from the glossary, acronyms or symbols) is already defined.

I hope that the information provided is enough to pinpoint the error. But feel free to ask for anything else you might need.

**Comment by petersaints** [comment]
For the time being I was able to edit the novathesis.cls file to force not to load the glossary and symbols:
```
\newcommand{\@loadglossaryfiles}{%
  \forcsvlist{\@loadglossaryfileifexists{glossaries}}{acronyms}%
}
```
But this is just a stopgap solution. I would need to better understand the template code before suggesting a proper solution.

**Comment by joaomlourenco** [comment]
Pedro,
Unfortunately version 5.x.y is still a bit unstable.  :(

That problem (and some others) is already fixed in the last version in GitHub (5.1.6]
